PowerShellTools.HostService/MainWindow.xaml.cs

```diff
@@ -26,9 +26,14 @@ public MainWindow()
         {
             InitializeComponent();
 
-            this.Visibility = System.Windows.Visibility.Hidden;
-            this.Topmost = true;
+            this.Visibility = System.Windows.Visibility.Visible;
             this.ShowInTaskbar = false;
+            this.Title = string.Format(
+                    PowerShellTools.Common.Constants.HostProcessWindowTitleFormat, 
+                    App.VsProcessId, 
+                    PowerShellTools.Common.Constants.PowerShellHostExeName);
+            this.Left = -1000;
+            this.Top = -1000;

```

Comment:
VSTO had to make a similar workaround once upon a time.   Guidance from the Windows team had some other specific constants to use.  Will dig up that information for you.